### DISCUSSION TITLE
SpeedForwarder (discussion)

### DISCUSSION THREAD

**Discussion by lostdusty** [FastForwardTeam/FastForward#657] [General]
You can discuss about #656 here.

**Comment by KanekoHato** [comment]
Hi can i just open a issue if i want to suggest a website ? 
also i kinda want to help with the localization to my language , is it possible ? 
i check the web to translate it but my flag isn't available

**Reply by lostdusty** [reply]
About new bypasses, no.
Yup it is, what is your language?

**Comment by Irmitya** [comment]
I was looking to bypass megaurl: megaurl.in/RitaWheeler
it redirected to another site: downfile.site
and then after doing a captcha, it redirects to some other site in some other language, filled with ads (ignored by ublock), for a 25 second wait.

I first saw universal-bypass which said it does megaurl and it's not maintained and redirects to this as a continuation, which apparently does not do megaurl.

**Reply by lostdusty** [reply]
Don't report broken bypasses here! Use the tab Issues for that.

**Comment by JRBros2346** [comment]
gplinks not working, i guess?

**Reply by lostdusty** [reply]
Don't report broken bypasses here! Use the tab "Issues" for that.

**Comment by CormacZ** [comment]
Hello, can you add bypass for " https://work.ink/ "?

**Reply by lostdusty** [reply]
Don't ask for bypasses here!

**Comment by pineconekr** [comment]
"link.codevn.net"  add please!

**Reply by lostdusty** [reply]
Don't ask for bypasses here!

**Comment by efedo2000** [comment]
work.ink and linkvertise.com please

**Reply by reashetyrr** [reply]
Yea we definitely have linkvertise in already!

**Comment by Cyberavater** [comment]
I think it'll be best to convert the project into a userscript at this point. That way it'll get auto updates and will be remove-proof even for google chrome.

**Reply by reashetyrr** [reply]
yes and no, with chromes manifest v3 userscript and unbundled scripts are not supported, making the whole project a userscript means chrome is completely useless

**Reply by Cyberavater** [reply]
Oh yes, I just found out about it right now.
https://github.com/Tampermonkey/tampermonkey/issues/644#issuecomment-1229203928
Maybe, it's time to switch the browser.

**Reply by lostdusty** [reply]
Chrome gave another year to MV3.. maybe they're changing their minds.

**Comment by reashetyrr** [comment]
guys just a psa, this discussion is not to tell us certain bypasses aint working, this is about the new manifest v3, you can discus stuff related to that just dont put messages like: "ad.fly please" "linkvertise is not working"  etc, we know that some bypasses aint working and we are working on porting the current ones to the new manifest v3 style we made

**Comment by godpow** [comment]
work.ink if possible please

**Reply by lostdusty** [reply]
Don't ask for bypasses here!

**Comment by rheznendra** [comment]
can't open nightly.link to download the zip file, seems dead for me
can you upload to other mirror?

**Reply by lostdusty** [reply]
Thinking about forking nightly.link & running our own instance

**Comment by N5F** [comment]
on the issues page it said that info about suggestions was at #656 but there was no info there, so fuck it: https://www.google.com/recaptcha/

**Comment by xodotLBO** [comment]
https://tinyurl.so/ doesn't get bypassed

**Reply by Hedreon** [reply]
> https://tinyurl.so/ doesn't get bypassed

Report broken bypasses in the issues tab (unless if it's not broken and you're requesting new bypasses. ...well, you just have to wait until they're finished)

**Comment by AnuAnu22** [comment]
https://cybertechng.com/

This website is not on your list
![Screenshot (18)](https://github.com/FastForwardTeam/FastForward/assets/129385032/be47acb2-b684-4ef8-803e-115e4ba1b7db)

**Reply by lostdusty** [reply]
Don't ask for bypasses here!

**Comment by lostdusty** [comment]
Speed Forward is out (It's the MV3 version of the extension)!
